.travis.yml

```diff
@@ -7,7 +7,7 @@ cache:
 before_script:
   - chmod +x scripts/docs
 script:
-  - npm run cover
+  - npm run cover && npm run lint
```

Comment:
I'll add them on separate lines. I think it's best to run both lint and tests on CI, so contributors get feedback on all errors (rather than needing to fix one at a time to see the results on the other).

If this works out, I'll rebase commits to match contribution guidelines... which I should have read earlier.